bindings/python/crocoddyl/core/action-base.hpp

```diff
@@ -54,6 +54,27 @@ class ActionModelAbstract_wrap : public ActionModelAbstract, public bp::wrapper<
   }
 
   boost::shared_ptr<ActionDataAbstract> default_createData() { return this->ActionModelAbstract::createData(); }
+
+  void quasiStatic(const boost::shared_ptr<ActionDataAbstract>& data, Eigen::Ref<Eigen::VectorXd> u,
+                   const Eigen::Ref<const Eigen::VectorXd>& x, const std::size_t maxiter, const double tol) {
+    if (boost::python::override createData = this->get_override("quasiStatic")) {
+      Eigen::VectorXd u_tmp =
+          bp::call<Eigen::VectorXd>(this->get_override("quasiStatic").ptr(), data, (Eigen::VectorXd)x, maxiter, tol);
+      if (static_cast<std::size_t>(u_tmp.size()) == nu_) {
+        u = u_tmp;
+      } else {
+        throw_pretty("Invalid argument: "
+                     << "u has wrong dimension (it should be " + std::to_string(nu_) + ")");
+      }
+      return;
+    }
+    return ActionModelAbstract::quasiStatic(data, u, x, maxiter, tol);
```

Comment:
Why are you returning ActionModelAbstract::quasiStatic(data, u, x, maxiter, tol) here, instead of u?

Reply:
The code within this function aims to allows inheritance in Python. If you see above, this function doesn't return any vector. Furthermore, this line is called only when `quasiStatic` has not been defined in Python.

I will mark this discussion as resolved.